core/storage/suggestion/gae_models.py

```diff
@@ -320,6 +320,49 @@ def get_translation_suggestions_in_review_with_exp_id(
             .fetch(feconf.DEFAULT_QUERY_LIMIT)
         )
 
+    @classmethod
+    def get_multiple_translation_suggestions_in_review(
+            cls, thread_ids: List[str]
+    ) -> List['GeneralSuggestionModel']:
+        """Returns translation suggestions which are in review from multiple
+        thread ids.
+
+        Args:
+            thread_ids: list(str). Suggestion ids of translations that are in
+                review.
+
+        Returns:
+            list(SuggestionModel). A list of translation suggestions in review
+            with given suggestion ids.
+        """
+        return cast(
+            List[GeneralSuggestionModel],
+            GeneralSuggestionModel.get_multi(thread_ids)
+        )
+
+    @classmethod
+    def get_translation_suggestions_in_review_ids_with_exp_id(
+            cls, target_ids: List[str]
+    ) -> List[str]:
+        """Returns ids of translation suggestions which are in review with
+        target ids.
+
+        Args:
+            target_ids: list(str). Exploration IDs matching the target ID of the
+                translation suggestions.
+
+        Returns:
+            list(str). A list of ids of translation suggestions in review
+            with target_id of exp_id.
+        """
+        suggestion_keys = cast(
+            List[datastore_services.Key],
```

Comment:
Do you need this cast? I thought @vojtechjelinek found a way to avoid this sort of thing in a PR some time back -- see https://github.com/oppia/oppia/pull/13865/files

Reply:
Actually I had not taken the latest from the upstream. And it was not necessary to cast and removed it.